kubejs/server_scripts/base/unification/unify_loot.js

```diff
@@ -1,196 +1,80 @@
-                        return ItemStack;
-                    }
-                }
-            }
-            console.error(
-                'LootJS Unification Loot Modifier triggered for Nuggets, but none of the required tags were found in the item: ' +
-                    ItemStack.getId()
-            );
-            return ItemStack;
-        })
+        .modifyLoot(Ingredient.of('#forge:nuggets'), (ItemStack) => {return unifyLoot("forge:nuggets", "nugget", ItemStack, null, true, 1)})
         // Storage Blocks
-        .modifyLoot(Ingredient.of('#forge:storage_blocks'), (ItemStack) => {
-            let iterator = ItemStack.getTags().iterator();
-            while (iterator.hasNext()) {
-                let tagString = iterator.next().location().toString();
-                if (tagString.startsWith('forge:storage_blocks/')) {
-                    let tagSubString = tagString.substring(21);
-                    if (Item.exists(`emendatusenigmatica:raw_${tagSubString}`)) {
-                        return Item.of(
-                            `emendatusenigmatica:raw_${tagSubString}`,
-                            ItemStack.getCount() * (Math.floor(Math.random() * 3) + 3)
-                        );
-                    } else if (Item.exists(`emendatusenigmatica:${tagSubString}_block`)) {
-                        return Item.of(`emendatusenigmatica:${tagSubString}_block`, ItemStack.getCount());
-                    } else {
-                        return ItemStack;
-                    }
-                }
-            }
-            console.error(
-                'LootJS Unification Loot Modifier triggered for Storage Blocks, but none of the required tags were found in the item: ' +
-                    ItemStack.getId()
-            );
-            return ItemStack;
-        })
-        //Gems
-        .modifyLoot(Ingredient.of('#forge:gems'), (ItemStack) => {
-            let iterator = ItemStack.getTags().iterator();
-            while (iterator.hasNext()) {
-                let tagString = iterator.next().location().toString();
-                if (tagString.startsWith('forge:gems/')) {
-                    let tagSubString = tagString.substring(11);
-                    if (Item.exists(`emendatusenigmatica:${tagSubString}_gem`)) {
-                        return Item.of(`emendatusenigmatica:${tagSubString}_gem`, ItemStack.getCount());
-                    } else {
-                        return ItemStack;
-                    }
-                }
-            }
-            console.error(
-                'LootJS Unification Loot Modifier triggered for Gems, but none of the required tags were found in the item: ' +
-                    ItemStack.getId()
-            );
-            return ItemStack;
-        });
+        .modifyLoot(Ingredient.of('#forge:storage_blocks'), (ItemStack) => {return unifyLoot("forge:storage_blocks", "block", ItemStack, null, true, ItemStack.getCount() * (Math.floor(Math.random() * 3) + 3))})
```

Comment:
If you omit `{ }` the return can be removed in lambda expressions

Reply:
o.o, this I actually didn't knew about, good to know!